xml/containers-support.xml

```diff
@@ -12,47 +12,63 @@
   <title>Compatibility and support plans</title>
  </info>
  <sect1 xml:id="sec-sle-container-support-matrix">
-  <title>&sle; container support matrix</title>
+  <title>Support information</title>
 
-  <para>
-   Consult the following support and compatibility matrix to make sure that the
-   desired host system and container combination is compatible and supported.
-  </para>
-
-  <informalfigure>
-   <mediaobject>
-    <imageobject role="fo">
-     <imagedata fileref="container_support_matrix.png" width="100%" format="PNG"/>
-    </imageobject>
-    <imageobject role="html">
-     <imagedata fileref="container_support_matrix.png" width="100%" format="PNG"/>
-    </imageobject>
-   </mediaobject>
-  </informalfigure>
+  <sect2 xml:id="sec-sle-container-non-sle-host-support">
+   <title>Support for &slsa; hosts</title>
+   <para>
+    Consult the following support and compatibility matrix to make sure that
+    the desired host system and container combination is compatible and
+    supported.
```

Comment:
"support and compatibility" and "compatible and supported" sounds redundant. Just make it "the following matrix".